ubsan_arangodb_suppressions.txt

```diff
@@ -15,6 +15,7 @@ null:3rdParty/rocksdb/file/random_access_file_reader.cc
 null:3rdParty/rocksdb/file/writable_file_writer.cc
 null:3rdParty/rocksdb/monitoring/perf_step_timer.h
 null:3rdParty/rocksdb/util/user_comparator_wrapper.h
+alignment:3rdParty/rocksdb/util/crc32c.cc
```

Comment:
Is this change intentional? It seems to have nothing to do with the actual bugfix.